README.md

```diff
@@ -94,9 +104,20 @@ deactivate
 module purge
 ```
 
-We highlight that the module files need to be loaded before every activation
+Please note the module files need to be loaded before every activation
 of the virtual environment.
 
+NumPy 2 users need to override the version requirements as follows:
+
+```sh
+sed -i "/numpy/d" requirements.txt
+python3 -m pip install -r requirements.txt "numpy==2.1" "pandas>=2"
+```
+
+Please note that for NumPy 2 to be supported, ESPResSo needs to be built
+against NumPy 2 too, otherwise it will complain about API version mismatch.
+Be aware the testsuite of ESPResSo 4.2 is known to be flaky with NumPy 2.
```

Comment:
I wanted to better separate all 3 alternatives (NumPy 1 users, cluster users, NumPy 2 users), using something similar to [sphinx tabs ](https://sphinx-tabs.readthedocs.io/en/latest/) like on the official Cython website ([example](https://cython.readthedocs.io/en/latest/src/tutorial/cython_tutorial.html#primes)), but the GitHub-flavored Markdown syntax doesn't support it. I've seen feature requests on GitHub support pages where people pointed out [R Markdown](https://bookdown.org/yihui/rmarkdown-cookbook/html-tabs.html) could do that, but it's unclear whether this is being worked on. Wrapping the alternatives into [collapsible sections](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/organizing-information-with-collapsed-sections) doesn't feel right to me, because this feature is typically used to hide "non-critical" information (long scripts or error logs) in bug reports to make them more readable.

Reply:
This was fixed by merging the NumPy 2 paragraph into the first paragraph.